deployments/helm/nvidia-dra-driver-gpu/templates/_helpers.tpl

```diff
@@ -43,36 +43,47 @@ Create chart name and version as used by the chart label.
 {{- end }}
 
 {{/*
-Common labels
+Standard labels: documented at
+https://helm.sh/docs/chart_best_practices/labels/
+Apply this to all high-level objects (Deployment, DaemonSet, ...).
+Pod template labels are included here to deliver name+instance.
 */}}
 {{- define "nvidia-dra-driver-gpu.labels" -}}
 helm.sh/chart: {{ include "nvidia-dra-driver-gpu.chart" . }}
-{{ include "nvidia-dra-driver-gpu.templateLabels" . }}
-{{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{ include "nvidia-dra-driver-gpu.templateLabels" . }}
 {{- end }}
 
 {{/*
-Template labels
```

Comment:
Slightly allergic to a thing named `templateLabels` being documented with "Template labels".